xml/cl.xml

```diff
@@ -6987,6 +6988,9 @@ server's OpenCL/api-docs repository.
             <require comment="cl_device_info">
                 <enum name="CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR"/>
             </require>
+            <require comment="cl_device_info">
+                <enum name="CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR"/>
+            </require>
```

Comment:
Nitpick: the header generation will look a little nicer if this enum is included in the same `<require>` block as `CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR`.

Reply:
Made the requested changes.